src/Code.gs

```diff
@@ -16,8 +16,7 @@ var BUILD = '1.0.6';
  */
 function onOpen(e) {
   addMenu();
-
-  if (e && e.authMode == ScriptApp.AuthMode.FULL) {
+  if (e && e.authMode == ScriptApp.AuthMode.LIMITED) {
```

Comment:
Whats the reason for this change?

Reply:
I had an issue which I could not resolve which I thought was related to this.
It should be reverted. My bad. :(